### ISSUE TITLE
Error in handleThirdPartyRoomMessage { [M_UNKNOWN: No row found]

### ISSUE THREAD

**Issue by MurzNN** [matrix-hacks/matrix-puppet-skype#11]
I got the error on receive incoming message in Skype room:
```
Error in handleThirdPartyRoomMessage { [M_UNKNOWN: No row found]
  errcode: 'M_UNKNOWN',
  name: 'M_UNKNOWN',
  message: 'No row found',
  data: { errcode: 'M_UNKNOWN', error: 'No row found' },
  httpStatus: 404 } { roomId: '<redacted>l',
  senderId: '<redacted>',
  senderName: '<redacted>',
  avatarUrl: 'https://avatar.skype.com/v1/avatars/<redacted>?auth_key=<redacted>',
  text: 'Не <redacted> ли случайно? )' } ﻿
```

**Comment by MurzNN** [comment]
This was be fist message from new user of room, next messages from this user received successfully. So maybe bridge receive user message earlier that new room member event?

**Comment by MurzNN** [comment]
Yes, when new user send me a message and no room exits in Matrix, first message is lost, because it received by Matrix puppet bridge before adding remote user to room event. Next messages received normally.

**Comment by MurzNN** [comment]
Same problem repeating when user is not added to Matrix room, but exits in Skype room. So now Skype room contains 4 users, but in Matrix room there are only 3 users and 1 missed, and all messages from missed user from Skype - generate `No row found` and missed in Matrix room. Will be good to check if Skype user exists in Matrix room, and force add it if absent.

**Comment by MurzNN** [comment]
Seems this is problem with specific contact: If this contact try to send me direct messages in new room, I see same error and no messages in Matrix. This contact have `-` in login, so nick is like `asd-qwe`, maybe the problem is in `-` symbol, I try to find other contacts with same symbol.

**Comment by MurzNN** [comment]
Seems the source of the problem is in #15 issue.

**Comment by twouters** [comment]
Should be fixed when matrix-puppet-bridge 1.15.4 is released.

**Comment by MurzNN** [comment]
Thanks, seems fixed - at now after upgrading and 4 days of testing - I don't see repeats of this error.
